### DISCUSSION TITLE
codespace error!

### DISCUSSION THREAD

**Discussion by ziqidella** [community/community#121046] [Codespaces]
### Select Topic Area

Bug

### Body

![Uploading error.jpg…]()
it's been like this for a couple of days, does anyone know how to solve this issue?

**Comment by ZFPuhi** [comment]
Hey,

Unfortunately the image that you have tried to upload is not showing, could you try to re-upload it or write what exactly you encounter as an issue/error for us to assist you in fixing it.

**Reply by ziqidella** [reply]
Hey, don't know why i keep having trouble uploading this picture..the error message reports this: ECONNABORTED: (signal aborted) canceled. It's been reporting the same error message for the past three days, and everything was fine before that.
Thanks for the help!
